doc/source/reference/commands/io_flush.rst

```diff
@@ -279,6 +279,81 @@ isn't specified:
 .. include:: ../../example/reference/commands/io_flush/recursive_default.log
 .. io_flush
 
+Since 9.0.2, ``recursive`` behavior is extended to flush not only target object
+and child objects, but also related objects. The related objects are:
+
+* A referenced table
+* A related index column (There is source column in target TABLE_NAME)
+* A table of related index column (There is source column in target TABLE_NAME)
+
+It is useful not to forget flushing related objects.
+
+For example, ``--recursive yes`` (default) is specified for ``TABLE_NAME``, this
+option executes equivalent to the following commands internally.
+
+- Flush table and its columns::
+
+    io_flush --target_name TABLE_NAME --recursive no
```

Comment:
I think if we execute this command, we can not but flush a table.
If we flush the table and its columns, we also execute a below command.

```
io_flush --target_name TABLE_NAME.COLUMN_NAME1 --recursive no
io_flush --target_name TABLE_NAME.COLUMN_NAME2 --recursive no
io_flush --target_name TABLE_NAME.COLUMN_NAME3 --recursive no
.
.
.
```

Reply:
Thanks!
I've fixed it.